### DISCUSSION TITLE
How to provide Go compiler flags correctly?

### DISCUSSION THREAD

**Discussion by Robert-M-Muench** [wailsapp/wails#2730] [Q&A]
I'm getting a VCS stamping error, and I just want to ignore it for the moment:

```
DEB | Using go webview2loader
Wails CLI v2.5.1

Executing: go mod tidy
  • Generating bindings:

  ERROR
          error obtaining VCS status: exit status 128
                Use -buildvcs=false to disable VCS stamping.

          exit status 1
```

But I don't understand how I could provide the `-buildvcs=false` flag. I tried a couple of combinations:

1. `wails dev -ldflags="buildvcs=false"`
2. `wails dev -ldflags="-buildvcs=false"`
3. `wails dev -ldflags -buildvcs=false`

What's the correct way to do that?

**Comment by Robert-M-Muench** [comment]
Anyone? This is blocking me for the moment.

**Reply by pidanou** [reply]
This worked for me:
`wails dev -appargs "'-env=dev'"`
